src/storage/src/sink/kafka.rs

```diff
@@ -244,7 +245,10 @@ impl TransactionalProducer {
         // Make the Kafka producer wait at least 10 ms before sending out MessageSets
         options.insert("queue.buffering.max.ms", format!("{}", 10));
         // Time out transactions after 60 seconds
-        options.insert("transaction.timeout.ms", format!("{}", 60_000));
+        options.insert(
+            "transaction.timeout.ms",
+            format!("{}", timeout_config.transaction_timeout.as_millis()),
+        );
```

Comment:
technically we already set this up in `create_with_context`, and are overriding it here...should we remove this line?